src/main/java/fm/last/commons/test/file/DataFolder.java

```diff
@@ -19,9 +19,9 @@
 import java.io.IOException;
 import java.net.URI;
 
-import org.junit.rules.MethodRule;
+import org.junit.rules.TestRule;
 
-public interface DataFolder extends MethodRule {
+public interface DataFolder extends TestRule {
```

Comment:
I wonder if we should go with a major version bump since we're changing a public interface (and public classes below). Does this require the end users of the rules to make any changes in their projects or is it transparent?

Reply:
I tested with the one of the tests from `apiary-extensions` and I didn't need to change anything. I think it should be transparent because nothing is changing from the user point of view, its just the internal stuff from JUnit thats changing

Reply:
OK, so let's make this a 6.1.0 release then (update pom accordingly please).

Reply:
done :))